zookeeper-server/src/main/java/org/apache/zookeeper/server/ZKDatabase.java

```diff
@@ -151,7 +158,24 @@ public ZKDatabase(FileTxnSnapLog snapLog) {
                 DEFAULT_COMMIT_LOG_COUNT);
             commitLogCount = DEFAULT_COMMIT_LOG_COUNT;
         }
-        LOG.info("{}={}", COMMIT_LOG_COUNT, commitLogCount);
+        try {
+            commitLogSize = Double.parseDouble(
```

Comment:
Makes sense to me and I think it's doable in this PR too since we're already touching this part. The parser is already merged and could be easily shared.